src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -425,7 +439,7 @@ const ExpandedNodeMemo: FC<ExpandedNodeMemoProps> = memo(
             </div>
             <textarea
               className={styles.comments}
-              value={comments}
+              defaultValue={comments}
```

Comment:
I think this should also remain value. Getting some unexpected behavior. Making this value and changing the comments prop back to `notes.note` means we will still see the comments in the box and the default text button will append correctly to the textbox
```suggestion
              value={comments}
```